browser/brave_news/sources.gni

```diff
@@ -10,6 +10,7 @@ brave_browser_brave_news_deps = []
 
 if (enable_brave_news) {
   brave_browser_brave_news_sources += [
+    "//brave/browser/brave_news/android/brave_news_controller_factory_android.cc",
```

Comment:
shouldn't we include that file only for android? `if (is_android)`

Reply:
yes please

Reply:
done